sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -143,6 +182,46 @@ poller = self.phone_number_client.begin_update_phone_number_capabilities(
 )
 ```
 
+### SipRoutingClient
+
+#### Retrieve SIP trunks and routes
+
+Get the list of currently configured trunks or routes.
+
+```python
+trunks = sip_routing_client.get_trunks()
+routes = sip_routing_client.get_routes()
+```
+
+#### Replace SIP trunks and routes
+
+Replace the list of currently configured trunks or routes with new values.
+
+```python
+sip_routing_client.set_trunks("<list of new trunks>")
+sip_routing_client.set_routes("<list of new routes>")
```

Comment:
Rather than using a string as the placeholder input - maybe it should be a list to avoid any confusion?
Better yet - an actual example of what this input might look like.

Reply:
Changed to provide an example of input.